src/components/DateInput/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./styles/styles.module.scss";
+
+const DateInput = ({ title, isRequired }) => {
+  let dateInput = (
+    <label className={styles.label}>
+      {title}
+      <input className={styles.input} type="date" />
+    </label>
+  );
+
+  if (isRequired) {
+    dateInput = (
+      <label className={styles.label}>
+        {title}
+        <input className={styles.input} type="date" required />
+      </label>
+    );
+  }
+
+  return dateInput;
+};
```

Comment:
Would this also work?
```suggestion
const DateInput = ({ title, isRequired }) => {
    return  (
      <label className={styles.label}>
        {title} {isRequired && "*"}
        <input className={styles.input} type="date" required={isRequired} />
      </label>
    );
};
```

Reply:
I didn't get what is happening here